zetaclient/chains/bitcoin/observer/observer.go

```diff
@@ -376,7 +376,7 @@ func (ob *Observer) PostGasPrice(ctx context.Context) error {
 	}
 
 	// #nosec G115 always positive
-	_, err = ob.ZetacoreClient().PostVoteGasPrice(ctx, ob.Chain(), feeRateEstimated, "100", uint64(blockNumber))
+	_, err = ob.ZetacoreClient().PostVoteGasPrice(ctx, ob.Chain(), feeRateEstimated, 0, uint64(blockNumber))
```

Comment:
lets add a comment to clarify why we are setting the 0 value here .

Reply:
https://github.com/zeta-chain/node/pull/2483/commits/4193e83b33a82eb2f701249f7fc1492a945e5987